ui/appui-react/src/appui-react.ts

```diff
@@ -175,7 +175,7 @@ export * from "./appui-react/statusbar/useDefaultStatusBarItems";
 
 export * from "./appui-react/statusfields/tileloading/TileLoadingIndicator";
 export * from "./appui-react/statusfields/ActivityCenter";
-export * from "./appui-react/statusfields/MessageCenter";
+export * from "./appui-react/layout/footer/message-center/MessageCenterField";
```

Comment:
Gerardas told me that we should keep field components in the `/statusfields` folder so you should move this back and probably add an inner folder for it like `/statusfield/messagecenter/`.